SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -85,24 +113,34 @@ if [ -s config_diff ]; then
 fi
 
 %build
-make KCFLAGS="-Wa,-mx86-used-note=no" bzImage VERBOSE=1 KBUILD_BUILD_VERSION="1" KBUILD_BUILD_HOST="CBL-Mariner" ARCH=%{arch} %{?_smp_mflags}
+%ifarch x86_64
+KCFLAGS="%{kcflags}" make VERBOSE=1 KBUILD_BUILD_VERSION="1" KBUILD_BUILD_HOST="CBL-Mariner" ARCH=%{arch} %{?_smp_mflags}
+%endif
+%ifarch aarch64
+make VERBOSE=1 KBUILD_BUILD_VERSION="1" KBUILD_BUILD_HOST="CBL-Mariner" ARCH=%{arch} %{?_smp_mflags}
+%endif
 
 %install
-install -vdm 700 %{buildroot}/boot
-
+D=%{buildroot}%{_datadir}/cloud-hypervisor
+install -D -m 644 %{image} $D/%{image_fname}
 %ifarch x86_64
-install -vm 600 arch/x86/boot/compressed/vmlinux.bin %{buildroot}/boot/vmlinux.bin
```

Comment:
This is an important change. Instead of installing the kernel image in /boot, we move that to /usr/share/cloud-hypervisor. Per Anatol, this is a more appropriate place and that's [how upstream spec does it](https://github.com/cloud-hypervisor/obs-packaging/blob/main/kernel/rpm/kernel-cloud-hypervisor-guest.spec#L80). So we also update the location [here](https://github.com/microsoft/CBL-Mariner/pull/5147/files#diff-7cd5459bb1e5a947c2a87599bc0cb4a92fcc7945e6c42d41991df822f80d9a46R248)